### PR TITLE
pre/postValidationFix changes 2

### PR DESCRIPTION
**Pull request by ammsalme**
* rename format to fix
* rename formatRecord to fixRecord

* split BIB_POSTVALIDATION_FIX_SETTINGS to BIB_POSTMERGE_FIX_SETTINGS and BIB_PREIMPORT_FIX_SETTINGS

* add stripF884s fix
** stripF884s fix removes extra duplicates of field 884 from the (merged) record. A 884 field is considered a duplicate, if it has organisation subfield $5 MELINDA and has identical contents than another field 884. Date subfield $g is excluded from the comparison. In case of duplicate the 884 field with the oldest data in the date subfield $g is kept.

* add stripF884s fix to BIB_POSTMERGE_FIX_SETTINGS

### CODE REVIEW

File: src/helpers/fix.js [hunk 1]
```diff
@@ -278,6 +367,7 @@ export const REPLACE_PREFIXES = [
 ];
 
 // If placed in config.js testing needs envs
```
Comment (natlibfi-jonollil) [inline src/helpers/fix.js:369]: Create file constants.js
and place const exports in there

File: src/helpers/fix.js [hunk 2]
```diff
@@ -128,10 +129,106 @@ export function formatRecord(record, settings = {}) {
     }
   }
 
+  function stripF884s(options) {
```
Comment (natlibfi-jonollil) [inline src/helpers/fix.js:132]: Nearly 100 line function could be better of in own file?

File: src/helpers/fix.spec.js [hunk 3]
```diff
@@ -29,16 +29,16 @@
 import {expect} from 'chai';
```
Comment (natlibfi-jonollil) [inline src/helpers/fix.spec.js:29]: Whole file should be converted to use fixugen

**Review by natlibfi-jonollil — CHANGES_REQUESTED** [review]

File: src/helpers/fix.js (hunk 1 above)
Comment (ammsalme) [reply src/helpers/fix.js]: Created a separate file for fix-constants

[64ab692](https://github.com/NatLibFi/melinda-rest-api-commons/pull/40/commits/64ab692f906645b369e9122173a1420a5b211f82)

File: src/helpers/fix.js (hunk 2 above)
Comment (ammsalme) [reply src/helpers/fix.js]: Split fix.js to several smaller files

[8e19fe5](https://github.com/NatLibFi/melinda-rest-api-commons/pull/40/commits/8e19fe5a6de1d9b82d2d1179298008b189148c18)

File: src/helpers/fix.spec.js (hunk 3 above)
Comment (ammsalme) [reply src/helpers/fix.spec.js:1]: Refactored tests for fix.js to use fixugen

[86b9ae0](https://github.com/NatLibFi/melinda-rest-api-commons/pull/40/commits/86b9ae07ece0a3deea7d55cc10f0661b3340141e) and [e23304a](https://github.com/NatLibFi/melinda-rest-api-commons/pull/40/commits/e23304a921e5971b57219bb51e2dce7abb078c42)

**Review by natlibfi-jonollil — APPROVED** [review]

**Pull request merged**